### DISCUSSION TITLE
[739630]Phasmophobia&恐鬼症 {本体}

### DISCUSSION THREAD

**Discussion by Minczm** [pjy612/SteamManifestCache#746] [Share]
### Steam 游戏ID

 ` 739630 ` 

### 密钥列表

```hash
739631;a0af61717bae449e5db4a1cc6eb68d7604ac3a8e05ff5226faa14b5be67d640c
```

### 清单附件

[739631_3760215442452486869.zip](https://github.com/user-attachments/files/16444583/739631_3760215442452486869.zip)

### 备注

email;2598719590@qq.com

**Comment by pjy612** [comment]
* #26 
* 为啥要加引号？

**Reply by Minczm** [reply]
已改

**Reply by Minczm** [reply]
密钥 的话 使用直接 ``` 进行，方便别人复制。
如
```
2238901;67cb73b183e7b5251669ae7065e69ae7e541ae57d56e19105e12323adcc2f42e
```

表示是把三引号看错了看成双引号

**Reply by pjy612** [reply]
那是之前 因为有人不会打 三反引号。。。 所以 专门写出来让人复制用的。
现在套了模板 直接写就行。
